indica/models/helike_spectroscopy.py

```diff
@@ -280,6 +280,10 @@ def _calculate_los_integral(self, calc_rho=False):
                 t=self.spectra.t,
                 calc_rho=calc_rho,
             )
+            # LOS integral removes NaNs so add them back
```

Comment:
Comment/suggestion:
I don't get this, and have tried it out and it seems this .where() is not required. 
Could it be legacy and could it be deleted?